core-bundle/contao/themes/flexible/styles/layout/data-container.pcss

```diff
@@ -4,6 +4,19 @@
     font-size: 1.1rem;
     display: flex;
     line-height: 22px;
+    align-items: center;
+}
+
+#main_headline .icon-favorite {
+    width: 24px;
```

Comment:
Just mentioning https://github.com/contao/contao/pull/8613 here as the height right now is set to the `line-height` of 22px, thus being 22px height 😊. Can however be fixed in a follow-up or maybe in 8613 if this PR is merged before it :)